internal/editor.go

```diff
@@ -82,7 +82,32 @@ func JSONFileEditor(src interface{}, dst interface{}) error {
 	}
 }
 
-// Get default editor
+// EditFile allows you to inplace edit a file until the editor returns success status
+func EditFile(file string) error {
+	editor, err := getEditor()
+	if err != nil {
+		return err
+	}
+
+	for {
```

Comment:
Why is this inside a loop?

Reply:
If there is an error code from the editor, it will try again... i don't think it's needed anymore though